cdap-runtime-ext-dataproc/src/main/resources/gce-dataproc.json

```diff
@@ -36,6 +38,7 @@
           "widget-type": "select",
           "label": "Region",
           "name": "region",
+          "description": "A region is a specific geographical location where you can run your resources",
```

Comment:
How about "A region is a specific geographical location where you can host resources such as static IP addresses. E.g. us-west1, us-central1, europe-west1. For more information, refer to https://cloud.google.com/compute/docs/regions-zones/"